website/snippets/_new-sl-setup.md

```diff
@@ -1,31 +1,54 @@
-You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
 
-- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
-    * Enterprise plan &mdash; Developer license with Account Admin permissions. Or Owner with a Developer license, assigned Project Creator, Database Admin, or Admin permissions.
-    * Team plan &mdash; Owner with a Developer license.
-    * If you are using a free trial dbt Cloud account, you are on a trial of the Team plan as an Owner, so you're good to go.
-- You must have a successful run in your new environment.
+Here's how to set up the Semantic Layer in dbt Cloud:
 
-Now that we've created and successfully run a job in your environment, you're ready to configure the semantic layer.
-
-1. Navigate to **Account Settings** in the navigation menu.
-2. Use the sidebar to select your project settings. Select the specific project you want to enable the Semantic Layer for.
-3. In the **Project Details** page, navigate to the **Semantic Layer** section, and select **Configure Semantic Layer**.
+### 1. Account settings and configuration
 
+- Navigate to **Account Settings** in the navigation menu.
+- On the **Settings** sidebar, select the specific project you want to enable the Semantic Layer for.
+- In the **Project details** page, navigate to the **Semantic Layer** section. Select **Configure Semantic Layer**  
 <Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
 
-4. In the **Set Up Semantic Layer Configuration** page, enter the credentials you want the Semantic Layer to use specific to your data platform.
+- In the **Semantic Layer & Credentials** page, select the deployment environment you want for the Semantic Layer and click **Save**. This provides administrators with the flexibility to choose the environment where the Semantic Layer will be enabled. <br />
+
+### 2. Add Semantic Layer credential and service token
+- <Expandable alt_header="Learn more about granular access control">
```

Comment:
do we have any guidance on best practices for users or admins? or maybe some example scenarios?

Reply:
we recommend building credentials like diff role based access. have a credential be a certain role that applies to diff users. but depends how they set up the access on how their teams are set up. recommend to create diff tokens/credentials based on what makes sense for your teams structure.